package.json

```diff
@@ -142,8 +142,10 @@
     "loader-utils": "^1.1.0",
     "ngtemplate-loader": "^1.3.1",
     "node-libs-browser": "^2.0.0",
+    "postcss-colorfix": "^0.0.3",
     "postcss-extend": "^1.0.5",
     "postcss-import": "^10.0.0",
+    "postcss-less": "^1.1.0",
```

Comment:
do we need `postcss-less`? It looks like postcss is processing plain CSS from `less-loader`